scripts/docstrings_checker.py

```diff
@@ -54,7 +54,7 @@ def get_setters_property_name(node):
     decorators = node.decorators.nodes if node.decorators else []
     for decorator in decorators:
         if (isinstance(decorator, astroid.Attribute) and
```

Comment:
Do you know why github is highlighting the closing parens in dark red?

Reply:
I have no idea. It's weird.

Reply:
Ah, try this -- I think it's recognizing decorator as a keyword rather than a variable name. Try changing decorators to decorator_nodes in line 54, and decorator to decorator_node. See if that works?

Reply:
Done.

@seanlip It works!